tests/e2e/operator.sh

```diff
@@ -303,6 +313,9 @@ usage() {
 	 "install": install only,
 	 "wait_for_stabilization": wait for CoCo pods to be stable
 	 "uninstall": uninstall the operator.
+	environment variables :
```

Comment:
I really like this section, could you please (in a separate commit) add the `RUNTIMECLASS`, which is already supported to be set?